Gemfile

```diff
@@ -49,7 +49,7 @@ end
 group :test do
   gem "capybara"
   gem "cuprite"
-  gem "faker"
```

Comment:
As mentioned in [the commit message](https://github.com/nhsuk/record-childrens-vaccinations/pull/257/commits/f959255e3a441dda91e8b2c5c033089c845d2990), this branch of Faker contains a change to allow the creation of an "alternate name" using an existing name,. I plan to contribute this upstream, but until then I'm pointing to my fork of Faker in case anyone wants to re-run the rake task that depends on it. This isn't on the critical path to delivery, and it is safe to point the gem back to mainstream Faker if needed.